docs/src/main/mdoc/typeclasses/bifoldable.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: docs
+title:  "Bifoldable"
+section: "typeclasses"
+source: "core/src/main/scala/cats/Bifoldable.scala"
+scaladoc: "#cats.Bifoldable"
+---
+
+# Bifoldable
+
+`Bifoldable` takes two type parameters and it's seen as a `Foldable` in both of these parameters.
```

Comment:
Sorry, I'm a little confused by this sentence, what does "seen as a `Foldable`" mean?

Reply:
I took that formulation from the existing `Bifunctor` [docs](https://typelevel.org/cats/typeclasses/bifunctor.html)

what about ..
```
`Bifoldable` takes two type parameters and is seen as working with 2 different `Foldables` to get one summary value.
```

Reply:
Ah, gotcha. I think it's the "seen as" part that I found confusing. The `Bifunctor` docs say it "_is_ a functor in both of these parameters".

Reply:
Any new thoughts about this one? :)

Reply:
Sorry, I forgot about this comm.

[Looked here](https://hackage.haskell.org/package/base-4.16.0.0/docs/Data-Bifoldable.html) for some inspiration

Maybe .. 
`Bifoldable` takes two type parameters and identifies foldable structures with two varieties of elements.

WDYT

Reply:
What about what it says in the scaladocs?
> A type class abstracting over types that give rise to two independent [cats.Foldable](https://typelevel.org/cats/api/cats/Foldable.html)s.

Reply:
so from the mdoc:
> Foldable type class instances can be defined for data structures that can be folded to a summary value.  

from the scaladocs:
> Bifoldable a type class abstracting over types that give rise to two independent [[cats.Foldable]]s.

hence I thought of:
> `Bifoldable[F[_,_]]` instances identify data structures with two independent `Foldable` that fold to the same summary value.
 
 
 note: I dropped explicitly stating it takes 2 type params, I think  the `Bifoldable[F[_,_]]` notation is clear enough at this point.

Reply:
Nice, I like this one the best! I appreciate you revising a few times. Thank you.